modules/abi-oci-resources-services.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-agent-based-installer.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="abi-oci-resources-services_{context}"]
+= OCI infrastructure resources and services
+
+You must create an {oci} environment on your virtual machine (VM) shape. By creating this environment, you can install {product-title} and deploy a cluster on infrastructure that supports a wide range of cloud options and strong security policies. Having prior knowledge of {oci} components can  help you with understanding the concept of {oci} resources and how you can configure them to meet your organizational needs.
+
+Before you create {oci} resources and services, you must configure an {oci} account with administrator privileges so that the account can manage {oci} resources and services, and hosts the {product-title} cluster.
+
+After you meet these prerequisites, you can create the following {oci} resources:
+
+* A compartment, which is a component where you can organize and isolate your cloud resources. After you create a compartment, Oracle automatically assigns an {ocid} to your organization’s account. An administrator can access all compartments tagged to your organization’s {oci} account.
+* A virtual cloud network (VCN). A compute instance, load balancer, and other resources need this network infrastructure to connect to each other over an internet connection.
+* A load balancer, so that you can provide automated traffic distribution on your VCN.
+* A network security group (NSG) in your VCN. You can use the NSG to establish advanced security rules for your network. You must locate the NSG in your compartment, so that certain groups can access network resources.
+* A dynamic group that hosts compute instances. After you create the dynamic group, you can then create a policy statement that defines rules for your cluster environment. This statement sets the precedent for each compute instance to join your {product-title} cluster as a self-managed node.
+* A policy statement, so that your administrator can grant access to your groups, users, or resources that operate in your network.
+* Three Domain Name System (DNS) records and then add the records to a DNS, so that Oracle’s edge-network can maintain your domain’s DNS queries.
```

Comment:
Neil's doc has this info
https://docs.google.com/document/d/1zYiZSydr-RkoX6UyYYdOU2QXE6foKcx0HkQwxObom4w/edit#heading=h.pj4nd423cwuc